website/frontend/src/components/pages/Meta/Meta.tsx

```diff
@@ -9,7 +10,7 @@ import { PageHeader } from 'components/blocks/PageHeader/PageHeader';
 import { useProgram } from 'hooks';
 
 type Params = {
-  programId: string;
+  programId: Hex;
```

Comment:
don't think that it's right, cuz programId clearly can be regular string and not hex